packages/accounts-password/password_client.js

```diff
@@ -208,8 +208,10 @@ Accounts.changePassword = function (oldPassword, newPassword, callback) {
  * @param {Function} [callback] Optional callback. Called with no arguments on success, or with a single `Error` argument on failure.
  */
 Accounts.forgotPassword = function(options, callback) {
-  if (!options.email)
-    throw new Error("Must pass options.email");
+  if (!options.email) {
+    callback(new Meteor.Error(400, "Must pass options.email"));
```

Comment:
`callback` is optional, you should check that it's available before using it.
Should we throw if there is no `callback`?